### DISCUSSION TITLE
表插入后查询失败failed to initialize storage reader

### DISCUSSION THREAD

**Discussion by flywuxu** [apache/doris#43054] [A - General / Q&A]
我在新表写入数据后,间隔一段时间后查询表就会出现如下问题

1105 - errCode = 2, detailMessage = (10.10.128.181)[CANCELLED]failed to initialize storage reader. tablet=1667386, res=[RUNTIME_ERROR]copy_column_data_by_selector nullable mismatch

	0#  doris::segment_v2::SegmentIterator::copy_column_data_by_selector(doris::vectorized::IColumn*, COW<doris::vectorized::IColumn>::mutable_ptr<doris::vectorized::IColumn>&, unsigned short*, unsigned short, unsigned long) at /home/zcp/repo_center/doris_release/doris/be/src/common/status.h:0
	1#  doris::Status doris::segment_v2::Segmen, Time: 0.070000s

**Comment by ixzc** [comment]
建议英文提问，如使用中文提问，可以去ask.selectdb.com中文技术论坛。里面有专人值班
